Fix indentation in deb package description

joseph-wakeling-sociomantic opened the pull request:
The preceding commit 6042c72a7f5e38ec9df1b8a41cee2753f0df45bf manually indented the body of the package description.  This is not necessary, as commands like `apt show` will handle this by themselves.

mathias-lang-sociomantic commented on pkg/libmxnet.pkg:
```diff
@@ -4,9 +4,9 @@ OPTS = dict(
- .
- It has been built with OpenCV and OpenMP support disabled.
+This package provides the core library exposing the MXNet C API.
+.
```
I suppose there is a reason for this lone dot ?

joseph-wakeling-sociomantic replied:
It seems to be a convention in deb-package descriptions, try `apt show [whatever]` to see what I mean.

If I had to guess, I'd say it is presumably so that there are no actual blank lines in the description, while allowing effective paragraphs.

joseph-wakeling-sociomantic replied:
See e.g. section 7.4 of https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics for an example.  (The indentation error in the original description patch was because I didn't realize the one-space indentation didn't have to be manual too.)

joseph-wakeling-sociomantic replied:
See also: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description

However, this is the _control file_ spec, so I'm going to just quickly check if I get the same result if I don't manually include the `.` but place a blank line in the description.

joseph-wakeling-sociomantic replied:
Yea, it's not needed in the `pkg` file.  I'm going to check out a couple of other things too and update accordingly.

joseph-wakeling-sociomantic commented:
Updated to remove the unnecessary `.`.  I've added a note on deb package description requirements to the commit message.

```diff
diff --git a/pkg/libmxnet.pkg b/pkg/libmxnet.pkg
index 3ca6a423..d05fd5aa 100644
--- a/pkg/libmxnet.pkg
+++ b/pkg/libmxnet.pkg
@@ -5,7 +5,7 @@ OPTS = dict(
     description = '''\
 Lightweight, portable deep learning library
 This package provides the core library exposing the MXNet C API.
-.
+
 It has been built with OpenCV and OpenMP support disabled.
 ''',
     url = 'https://github.com/dmlc/mxnet',
```
